### PR TITLE
Usability tweaks

### PR DESCRIPTION
**Pull request by t-young31**
- Update for [autodE](https://github.com/duartegroup/autodE) v1.1
- Adds Gaussian 09 and Gaussian 16 reference method calculations
- Tweaks the default SOAP expansion order (6->8)

### CODE REVIEW

File: gaptrain/calculators.py [hunk 1]
```diff
@@ -62,11 +62,21 @@ def run_autode(configuration, max_force=None, method=None, n_cores=1, kwds=None)
...
-    
-    if method.name == 'orca' and GTConfig.orca_keywords is None and kwds is None:
-        raise ValueError("For ORCA training GTConfig.orca_keywords must be"
-                         " set. or this function called with kwds e.g. "
-                         "GradientKeywords(['PBE', 'def2-SVP', 'EnGrad'])")
+
+    if method.name == 'orca':
+        if GTConfig.orca_keywords is None and kwds is None:
+            raise ValueError("For ORCA training GTConfig.orca_keywords must be"
+                             " set. or this function called with kwds e.g. "
```
Comment (tristan-j-wood) [inline gaptrain/calculators.py:69]: add space to line above

File: gaptrain/configurations.py [hunk 2]
```diff
@@ -622,7 +655,7 @@ def optimise(self, method_name, max_force):
 
     def single_point(self, method_name):
         """Run parallel single points"""
-        assert method_name in ('dftb', 'gpaw', 'orca', 'xtb', 'cp2k')
+        assert method_name in ('dftb', 'gpaw', 'orca', 'g09', 'xtb', 'cp2k')
```
Comment (tristan-j-wood) [inline gaptrain/configurations.py:658]: no g16?

**Review by tristan-j-wood — COMMENTED** [review]
Looks mostly fine

File: gaptrain/calculators.py (hunk 1 above)
Comment (t-young31) [reply gaptrain/calculators.py:69]: sadly this line is on 79 characters

**Pull request merged**